pkg/api/types.go

```diff
@@ -3109,6 +3109,8 @@ const (
 	NodeMemoryPressure NodeConditionType = "MemoryPressure"
 	// NodeDiskPressure means the kubelet is under pressure due to insufficient available disk.
 	NodeDiskPressure NodeConditionType = "DiskPressure"
+	// NodeCPUPressure means the kubelet is under pressure due to insufficient available cpu.
```

Comment:
Please see the [other comment thread in this PR](https://github.com/kubernetes/kubernetes/pull/49186#discussion_r134920816) where the implications are explained a little more clearly.